package.json

```diff
@@ -81,9 +81,10 @@
         "react-dom": "18"
     },
     "dependencies": {
+        "@prezly/content-renderer-react-js": "^0.30.1",
```

Comment:
The package is a dependency of itself? :thinking:

Reply:
Oops, that was a mistake let me update :(